management/src/main/java/org/kaazing/gateway/management/context/ManagementGatewayListener.java

```diff
@@ -69,8 +92,8 @@ public void startingService(ServiceContext serviceContext) {
     }
 
     @Override
-    public void stoppedGateway() {
```

Comment:
Looks as if you removed the stoppedGateway method. Are you sure we don't need it?